src/dailies.ts

```diff
@@ -180,12 +180,8 @@ function voterSetup(): void {
       })
     );
 
-    const initiativeValue = sum(
-      Array.from(availableInitiatives.values())
-        .sort((a, b) => b - a)
-        .slice(1),
-      (val) => 2 * val
-    );
+    const initiativeValue = 2 * Math.max(...availableInitiatives.values());
```

Comment:
this is more readable. good job. my one thing here would be that the actual list of valuations is kind of mediocre but it's fine; a few comments explaining would be nice. in particular, i think you could make a justification that famxp is a bit higher because it helps with goose leveling for the 50k meat skill.